client/src/language/EmbeddedLanguageDocsManager.ts

```diff
@@ -56,6 +56,21 @@ export default class EmbeddedLanguageDocsManager {
     })
   }
 
+  async deleteEmbeddedLanguageDocsFolder (): Promise<void> {
+    logger.debug(`Delete embedded language documents folder: ${this.embeddedLanguageDocsFolder}.`)
+    await new Promise<void>((resolve) => {
+      if (this.embeddedLanguageDocsFolder === undefined) {
+        return
+      }
+      fs.rm(this.embeddedLanguageDocsFolder, { recursive: true }, (err) => {
+        if (err !== null) {
+          logger.error(`Failed to delete embedded language documents folder: ${err as any}`)
```

Comment:
reject here? (not needed, since we don't handle it, but better semantic)

Reply:
Done.

I was thinking we did not care, so not rejecting would prevent VS Code from displaying an annoying error. But I changed my mind. This is likely a real system error, and the user should know.